Sheppard/build.gradle

```diff
@@ -1,10 +1,10 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
+    compileSdkVersion 30
     defaultConfig {
         applicationId "com.truex.sheppard"
-        minSdkVersion 21
+        minSdkVersion 25
         targetSdkVersion 26
```

Comment:
The `targetSdkVersion` and `compileSdkVersion` should be the same or we may run into issues with unsupported APIs at runtime.

Reply:
ok i'll just update both to 30, ok?